src/main/java/net/rptools/maptool/client/swing/ButtonKind.java

```diff
@@ -0,0 +1,112 @@
+  NEXT           ("NEXT"           , ButtonPanel.OTHER_BUTTON      , "Button.next"          , "Button.next.mnemonic"          ),
+  NETWORKING_HELP("NETWORKING_HELP", ButtonPanel.HELP              , "Button.networkingHelp", "Button.networkingHelp.mnemonic"),
+  NO             ("NO"             , ButtonPanel.CANCEL_BUTTON     , "Button.no"            , "Button.no.mnemonic"            ),
+  OK             ("OK"             , ButtonPanel.AFFIRMATIVE_BUTTON, "Button.ok"            , "Button.ok.mnemonic"            ),
+  OPEN           ("OPEN"           , ButtonPanel.AFFIRMATIVE_BUTTON, "Button.open"          , "Button.open.mnemonic"          ),
+  PRINT          ("PRINT"          , ButtonPanel.OTHER_BUTTON      , "Button.print"         , "Button.print.mnemonic"         ),
+  REFRESH        ("REFRESH"        , ButtonPanel.OTHER_BUTTON      , "Button.refresh"       , "Button.refresh.mnemonic"       ),
+  REPLACE        ("REPLACE"        , ButtonPanel.OTHER_BUTTON      , "Button.replace"       , "Button.replace.mnemonic"       ),
+  RESET          ("RESET"          , ButtonPanel.OTHER_BUTTON      , "Button.reset"         , "Button.reset.mnemonic"         ),
+  RETRY          ("RETRY"          , ButtonPanel.OTHER_BUTTON      , "Button.retry"         , "Button.retry.mnemonic"         ),
+  REVERT         ("REVERT"         , ButtonPanel.OTHER_BUTTON      , "Button.revert"        , "Button.revert.mnemonic"        ),
+  SAVE           ("SAVE"           , ButtonPanel.AFFIRMATIVE_BUTTON, "Button.save"          , "Button.save.mnemonic"          ),
+  SAVE_AS        ("SAVE_AS"        , ButtonPanel.AFFIRMATIVE_BUTTON, "Button.saveAs"        , "Button.saveAs.mnemonic"        ),
+  SHOW_DETAILS   ("SHOW_DETAILS"   , ButtonPanel.OTHER_BUTTON      , "Button.showDetails"   , "Button.showDetails.mnemonic"   ),
+  STOP           ("STOP"           , ButtonPanel.OTHER_BUTTON      , "Button.stop"          , "Button.stop.mnemonic"          ),
+  UPDATE         ("UPDATE"         , ButtonPanel.OTHER_BUTTON      , "Button.update"        , "Button.update.mnemonic"        ),
+  YES            ("YES"            , ButtonPanel.AFFIRMATIVE_BUTTON, "Button.yes"           , "Button.yes.mnemonic"           ),
+  MOVE_UP        ("MOVE_UP"        , ButtonPanel.OTHER_BUTTON      , "Button.moveUp", "Button.moveUp.mnemonic"        ),
+  MOVE_DOWN      ("MOVE_DOWN"      , ButtonPanel.OTHER_BUTTON      , "Button.moveDown", "Button.moveDown.mnemonic"      );
+  //  @spotless:on
+  final String name;
+  final String buttonPanelButtonType;
+  final String i18nKey;
+  final String i18nMnemonicKey;
+
+  ButtonKind(String name, String buttonPanelButtonType, String i18nKey, String i18nMnemonicKey) {
+    this.name = name;
+    this.buttonPanelButtonType = buttonPanelButtonType;
+    this.i18nKey = i18nKey;
+    this.i18nMnemonicKey = i18nMnemonicKey;
+  }
+
+  public int getMnemonicKey() {
```

Comment:
Might as well move this look up into the constructor so it's only calculated once and will be guaranteed to be consistent during the runtime.